src/crudeditor-lib/rootSaga.js

```diff
@@ -20,15 +20,17 @@ import {
   VIEW_ERROR
 } from './common/constants';
 
-const initializeViewSagas = {
-  [VIEW_SEARCH]: searchViewScenario,
-  [VIEW_CREATE]: createViewScenario,
-  [VIEW_EDIT]: editViewScenario,
-  [VIEW_SHOW]: showViewScenario,
-  [VIEW_ERROR]: errorViewScenario
-};
 
 export default function*(modelDefinition) {
+  const initializeViewSagas = (ops => ({
```

Comment:
Build the object directly instead of indirectly via a function.

Reply:
Done.